core/src/banking_stage/vote_worker.rs

```diff
@@ -412,7 +412,15 @@ impl VoteWorker {
         ProcessTransactionsSummary {
             reached_max_poh_height,
             transaction_counts: total_transaction_counts,
-            retryable_transaction_indexes,
+            // vote thread does not care about reason for retryable transactions
+            retryable_transaction_indexes: retryable_transaction_indexes
+                .into_iter()
+                .map(|index_kind| match index_kind {
+                    RetryableIndexKind::AccountInUse(index)
+                    | RetryableIndexKind::BlockLimits(index)
```

Comment:
I'm okay punting on it, but should votes also not waste time retrying on block limits?

Reply:
probably not; but less concerning for votes which do not actually hit the account limits which is what this PR is mostly concerned about.
if we hit block-limits it doesn't matter too much if we spin because the block is already full.
when we hit account limits and retry loop its an issue because the block may not be full yet